libraries/botbuilder-core/botbuilder/core/bot_framework_adapter.py

```diff
@@ -1109,7 +1234,7 @@ def key_for_connector_client(service_url: str, app_id: str, scope: str):
             self._is_emulating_oauth_cards = True
 
         app_id = self.__get_app_id(context)
-        scope = context.turn_state[BotAdapter.BOT_OAUTH_SCOPE_KEY]
+        scope = context.turn_state.get(BotAdapter.BOT_OAUTH_SCOPE_KEY)
```

Comment:
what is the difference between using [] and get() ?

Reply:
get does not raises KeyError if element does not exists, just returns None